php-checks/src/main/java/org/sonar/php/checks/DeprecatedPredefinedVariablesUseCheck.java

```diff
@@ -42,11 +42,18 @@ private void checkVariable(SyntaxToken variable) {
     String name = variable.text();
 
     if (CheckUtils.PREDEFINED_VARIABLES.containsKey(name)) {
-      String message = String.format(MESSAGE, name, CheckUtils.PREDEFINED_VARIABLES.get(name));
-      context().newIssue(this, variable, message);
+      String replacement = CheckUtils.PREDEFINED_VARIABLES.get(name);
+      raiseIssue(variable, name, replacement);
+    } else if ("$php_errormsg".equals(name)) {
```

Comment:
@pynicolas why can't we extend the list of `PREDEFINED_VARIABLES`, it looks like `$php_errormsg` is predefined variable as well (http://php.net/manual/en/reserved.variables.php)
of if I missing some point, may be `CheckUtils.PREDEFINED_VARIABLES` should be renamed

Reply:
@vilchik-elena I renamed `PREDEFINED_VARIABLES` to `SUPERGLOBALS_BY_OLD_NAME` in 646a7ed7cf84545523fe2b7289bee58742d04b07